docfiles/apptracking.html

```diff
@@ -29,14 +30,29 @@
             }
         });
 
-        var scriptIdRegex = /(?:S?\d{5}-\d{5}-\d{5}-\d{5})|(?:_[0-9a-zA-Z]{12})/g;
         // Check if the current page contains a share URL
         function isScriptPage(url) {
```

Comment:
this function looks not used? can probably just delete and move the regex into scrubUrl then